tests/meson.build

```diff
@@ -86,6 +86,15 @@ if gtest_dep.found()
   )
 
   test('unittest_plugins', unittest_plugins, args: ['--gst-plugin-path=..'])
+
+  # Run unittest_src_iio
+  unittest_src_iio = executable('unittest_src_iio',
```

Comment:
[suggestion] 
to run testcase, insert tc into unittest_plugins
or update /debian/rules and /packaging/nnstreamer.spec

Reply:
updated debian/rules and packaging/nnstreamer.spec for testcases